code/datums/ai/movement/ai_movement_basic_avoidance.dm

```diff
@@ -6,12 +6,19 @@
 ///Put your movement behavior in here!
 /datum/ai_movement/basic_avoidance/process(delta_time)
 	for(var/datum/ai_controller/controller as anything in moving_controllers)
+
+		var/atom/movable/movable_pawn = controller.pawn
+
+		// Check if this controller can actually run, so we don't chase people with corpses
+		if(!controller.able_to_run())
```

Comment:
shouldnt this be after the cooldown code?

Reply:
I can shift it - I wasn't sure if a hard stop to movement outside of the cooldowns would be better